src/LmdbCache.ts

```diff
@@ -8,13 +8,16 @@ import {
   SortKeyCacheResult,
   PruneStats
 } from 'warp-contracts';
-import { RootDatabase, open } from 'lmdb';
+import { RootDatabase, open, RangeOptions } from 'lmdb';
 
 export class LmdbCache<V = any> implements SortKeyCache<V> {
   private readonly logger = LoggerFactory.INST.create('LmdbCache');
 
   private readonly db: RootDatabase<V, string>;
 
+  private maxEntriesPerContract = 100
```

Comment:
it would be good to make these values configurable via constructor, with some default values.

Reply:
also...imho it would be best to leave one parameter..simply max entries per contract..

Reply:
It should be configurable via constructor, as an additional (optional) parameter  to the standard 'cacheOptions'

Reply:
Done